content/en/docs/reference/config/config-status/index.md

```diff
@@ -0,0 +1,105 @@
+status of a virtual service, use the following command:
+
+{{< text bash >}}
+$ kubectl get virtualservice service-name
+{{< /text >}}
+
+In the output, the `status` field contains several nested fields with details
+about the process of propagating configuration changes through the mesh. A
+child field for the condition status, also named `status`, has a value of `True`
+when Istio applied a configuration successfully. A value of `False` indicates
+there was an error.
+
+{{< text yaml >}}
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-26T22:06:34Z"
+    message: "61/122 complete"
+    reason: "stillPropagating"
+    status: "False"
+    type: Reconciled
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-26T22:06:56Z"
+    message: "1 Error and 1 Warning found. See validationMessages field for details"
+    reason: "errorsFound"
+    status: "False"
+    type: PassedAnalysis
+  validationMessages:
+  - code: IST0101
+    level: Error
+    message: 'Referenced gateway not found: "bogus-gateway"'
+  - code: IST0102
+    level: Warn
+    message: 'mTLS not enabled for virtual service'
+{{< /text >}}
+
+## The `type` and `status` field workflow
+
+The `type` field indicates the configuration condition. It can have the
+following values:
+
+* `PassedAnalysis`
+* `Reconciled`
+
+When you apply a configuration, the `type` field immediately sets to
+`Reconciled` and `status` to `False`, which blocks further commands until
+`status` equals `True` or the time out period expires. The `status` field might
+transition beyond `False` instantaneously, depending on the speed of the
+cluster.
+
+In parallel, the `PassedAnalysis` field is set to `True` or `False` to indicate
+the outcome of configuration validation. `PassedAnalysis` is an informational
+field only, meaning that it does not block the application of an invalid
+configuration. For example, it is possible for the status to indicate that
+validation failed, but applying the configuration was successful. This means
+Istio was able to set the new configuration, but the configuration was invalid,
+likely due to a syntax error or similar problem.
+
+The `type` field works in conjunction with the `status` field, indicating what
+part of the configuration process was successful or not. For example, when the
+`type` field value is `Reconciled`, it means that `status` indicates the outcome
+of applying the configuration. Similarly, when `type` equals `PassedAnalysis`,
+it indicates the results of validating the syntax of the CRD you provided to the
+`kubectl apply` command.
+
+## The `validationMessages` field
+
+In case of a validation failure, check the `validationMessages` field for
+more information. The `validationMessages` field has details about the validation
+process, such as error messages indicating that Istio cannot apply the
+configuration, and warning or informational messages that did not result in an
```

Comment:
We should probably add some documentation here regarding the structure of validation messages.  @zerobfd  can you help here?

Reply:
I don't think I ever got enough momentum to get the proposed spec into the API repo, so there's not really a guaranteed structure per se. We could link to https://istio.io/docs/reference/config/analysis/ here so that readers can see the various codes.

If everyone feels that not having a formal structure is insufficient here, I can see about finding time to re-start adding the structure, but I don't think it'll happen by 1.6.

Reply:
I am working in this direction, and I expect it will have a 1.7 eta.  Collaboration welcome.  In the meantime, a short description of what is written today would be valuable.

Reply:
Fixed. Added a link to the config reference for now.